spec/lib/vcr/cassette/serializers_spec.rb

```diff
@@ -17,7 +17,7 @@ class Cassette
               result = serializer.serialize("a" => string)
               serializer.deserialize(result)
             }.to raise_error(error_class, /preserve_exact_body_bytes/)
-          end unless (RUBY_INTERPRETER == :rubinius && RUBY_VERSION =~ /^1.9/)
+          end unless (RUBY_INTERPRETER == :rubinius)
```

Comment:
I think the original meant that the spec was not skipped on Rubinius targeting Ruby 2.0 and up. So the whole conditional can be removed.